xds/src/main/java/io/grpc/xds/WeightedTargetLoadBalancer.java

```diff
@@ -131,20 +135,28 @@ private void updateOverallBalancingState() {
     List<WeightedChildPicker> childPickers = new ArrayList<>();
 
     ConnectivityState overallState = null;
+    Status errorStatus = Status.UNAVAILABLE;
```

Comment:
It looks like this `Status.UNAVAILABLE` should never actually be used. Initialize the Status to null instead. It is normally wrong to create a "bare" status with just a code; almost universally it should have at least a description as well.